sdl2promela/sdl/__init__.py

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env python3
```

Comment:
I'm not sure if this is an executable script

Reply:
No, I just followed the convention used in opengeode, but after some reading, it is ok to leave this empty